### PR TITLE
Update Ludwig README

### PR DESCRIPTION
**Pull request by justinxzhao**
The README has a larger focus on the unique value that Ludwig brings, with ample pointers to official docs for diving deeper into specific functionalities.

To see the README visually (which may be easier to review), look at the branch directly: https://github.com/ludwig-ai/ludwig/tree/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,258 +16,467 @@
...
-Simple commands can be used to train models both locally and in a distributed way, and to use them to predict new data.
+Ludwig is a data-driven declarative ML framework hosted by the LF AI & Data Foundation that makes it easy to define
+deep learning pipelines for many types of tasks with a simple and flexible data-driven configuration system.
 
-A programmatic API is also available to use Ludwig from Python.
-A suite of visualization tools allows you to analyze models' training and test performance and to compare them.
+![img](https://raw.githubusercontent.com/ludwig-ai/ludwig-docs/ludwig05/docs/images/ludwig_legos.gif)
 
-Ludwig is built with extensibility principles in mind and is based on datatype abstractions, making it easy to add support for new datatypes as well as new model architectures.
+Ludwig supports many different combinations of input and output types that solve many different machine learning tasks.
```
Comment (jimthompson5802) [inline README.md:26]: This is proably a nit...replace the second occurrence of "many different" with "diverse"...something like this.

> Ludwig supports many different combinations of input and output types that solve diverse machine learning tasks.

File: README.md [hunk 2]
```diff
@@ -16,268 +16,188 @@
...
-an additional datatype that is not currently supported by adding a
-datatype-specific implementation of abstract classes that contain functions to
-preprocess the data, encode it, and decode it.
+- [Text Classification](https://ludwig-ai.github.io/ludwig-docs/latest/examples/text_classification)
+- [Named Entity Recognition Tagging](https://ludwig-ai.github.io/ludwig-docs/latest/examples/ner_tagging)
+- [Natural Language Understanding](https://ludwig-ai.github.io/ludwig-docs/latest/examples/nlu)
+- [Machine Translation](https://ludwig-ai.github.io/ludwig-docs/latest/examples/machine_translation)
+- [Chit-Chat Dialogue Modeling through seq2seq](https://ludwig-ai.github.io/ludwig-docs/latest/examples/seq2seq)
+- [Sentiment Analysis](https://ludwig-ai.github.io/ludwig-docs/latest/examples/sentiment_analysis)
+- [Image Classification](https://ludwig-ai.github.io/ludwig-docs/latest/examples/image_classification)
```
Comment (jimthompson5802) [inline README.md:174]: FYI....When the PR for the reorganized table of contents for the Examples is submitted, this example will not be there.  This is basically a duplicate of the MNIST example with just different category labels.  I decided to remove it because it does not show anything different than the MNIST example.

Comment (justinxzhao) [reply README.md]: Removed/reorganized!

File: README.md [hunk 3]
```diff
@@ -14,270 +14,347 @@
...
-If your dataset also contains ground truth values of the target outputs, you can compare them to the predictions obtained from the model to evaluate the model performance.
+Ludwig takes care of the engineering complexity of deep learning out of the box, enabling research scientists to focus on building models at the highest level of abstraction.
 
-```
-ludwig evaluate --dataset path/to/data.csv --model_path /path/to/model
-```
+Data preprocessing, hyperparameter optimization, device management, and distributed training for newly registered `torch.nn.Module` models come completely free.
 
-This will produce evaluation performance statistics that can be visualized by the `visualize` tool, which can also be used to compare performances and predictions of different models, for instance:
+## Easily build your benachmarks
```
Comment (jppgks) [inline README.md:275]: Typo: 
```suggestion
## Easily build your benchmarks
```

File: README.md [hunk 4]
```diff
@@ -14,270 +14,347 @@
...
-Ludwig also provides a simple programmatic API that allows you to train or load a model and use it to obtain predictions on new data:
+## Low-code interface for state-of-the-art models, including pre-trained Huggingface Transformers
+
+Ludwig also natively integrates with pre-trained models, such as the ones available in [Huggingface Transformers](https://huggingface.co/docs/transformers/index). Users can choose from a vast collection of state-of-the-art pre-trained PyTorch models to use without needing to write any code at all. For example, training a BERT-based sentiment analysis model with Ludwig is as simple as:
+
+```shell
+ludwig train --dataset sst5 -–config_str “{input_features: [{name: sentence, type: text, encoder: bert}], output_features: [{name: label, type: category}]}”
+```
+
+## Low-code interface for autoML
```
Comment (jppgks) [inline README.md:297]: To capitalize Auto to be consistent with other mentions of AutoML:
```suggestion
## Low-code interface for AutoML
```
